"{\"data\": [\"'{\\\"error\\\": {\\\"exception\\\": \\\"\\\\'function\\\\' object is not iterable\\\", \\\"message\\\": \\\"Traceback (most recent call last):\\\\\\\\n  File \\\\\\\\\\\"/usr/local/lib/python3.10/site-packages/mage_ai/data_preparation/models/widget/__init__.py\\\\\\\\\\\", line 408, in _execute_block\\\\\\\\n    outputs = self.post_process_variables(\\\\\\\\n  File \\\\\\\\\\\"/usr/local/lib/python3.10/site-packages/mage_ai/data_preparation/models/widget/__init__.py\\\\\\\\\\\", line 144, in post_process_variables\\\\\\\\n    values = [v for v in arr if v is not None and not np.isnan(v)]\\\\\\\\nTypeError: \\\\'function\\\\' object is not iterable\\\\\\\\n\\\"}, \\\"columns\\\": [\\\"id\\\", \\\"company_name\\\", \\\"contact_name\\\", \\\"contact_title\\\", \\\"address\\\", \\\"city\\\", \\\"region\\\", \\\"postal_code\\\", \\\"country\\\", \\\"phone\\\", \\\"fax\\\", \\\"order_id\\\", \\\"customer_id\\\", \\\"employee_id\\\", \\\"order_date\\\", \\\"required_date\\\", \\\"shipped_date\\\", \\\"ship_via\\\", \\\"freight\\\", \\\"ship_name\\\", \\\"ship_address\\\", \\\"ship_city\\\", \\\"ship_region\\\", \\\"ship_postal_code\\\", \\\"ship_country\\\"]}'\", \"\"], \"error\": null, \"execution_state\": null, \"metadata\": {}, \"msg_id\": \"3a10a0b6-0d8fbe52a38e4e37ff91e92b_1_29\", \"msg_type\": \"execute_result\", \"type\": \"text/plain\", \"block_type\": \"chart\", \"pipeline_uuid\": null, \"uuid\": \"bar_chart_for_throbbing_sky_1697988801064\"}"